docs/src/reference/torch/index.rst

```diff
@@ -0,0 +1,91 @@
+.. _torch-api-reference:
+
+TorchScript API reference
+=========================
+
+.. py:currentmodule:: equistore.torch
+
+We provide a special PyTorch C++ extension exporting all of the core equistore
+types in a way compatible with the TorchScript compiler, allowing users to save
+and load models based on equistore everywhere TorchScript is supported. In
+particular, this allow to define, train and save a model from Python, and then
+load it with pure C++ code, without requiring a Python interpreter. Please refer
+to the :ref:`installation instructions <install-torch-script-lib>` to know how
+to install the Python and C++ sides of this library.
+
+The classes and functions in the TorchScript API are kept as close as possible
+to the classes and functions of the pure Python API, with the explicit goal that
+changing from
+
+.. code-block:: python
+
+    import equistore
+    from equistore import TensorMap, TensorBlock, Labels
+
+to
+
+.. code-block:: python
+
+    import equistore.torch as equistore
+    from equistore.torch import TensorMap, TensorBlock, Labels
+
+is 80% of the work required to make a model developed in Python --- without
```

Comment:
If you only use equistore operations from core and equistore-operations, that will be True, but if you have custom equistore operations then not.

```suggestion
makes all the handling of equistore objects through equistore-operations compatible with TorchScript.
```